umm-spec-lib/src/cmr/umm_spec/migration/version/core.clj

```diff
@@ -15,11 +15,7 @@
    [cmr.umm-spec.migration.organization-personnel-migration :as op]
    [cmr.umm-spec.migration.related-url-migration :as related-url]
    [cmr.umm-spec.migration.spatial-extent-migration :as spatial-extent]
-   [cmr.umm-spec.migration.version.collection]
```

Comment:
I believe these imports are needed for loading in the concrete `defmethod` implementations.

Reply:
You're right! Added back.